### PR TITLE
Target .NET Standard 2.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: build/common.props [hunk 1]
```diff
@@ -18,8 +18,7 @@
     <PackageReference Include="Internal.AspNetCore.Sdk" Version="$(InternalAspNetCoreSdkVersion)" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
-    <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
+    <PackageReference Include="NETStandard.Library.NETFramework" Version="$(NETStandardLibraryNETFrameworkVersion)" />
   </ItemGroup>
-
```
Comment (pranavkm) [inline build/common.props]: Keep this

File: samples/RoutingSample.Web/RoutingSample.Web.csproj [hunk 2]
```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
```
Comment (JunTaoLuo) [inline samples/RoutingSample.Web/RoutingSample.Web.csproj]: We shouldn't depend on common in samples. Import dependency.props instead and add the explicity reference to NETStandard.Library.NETFramework

Comment (pranavkm) [inline samples/RoutingSample.Web/RoutingSample.Web.csproj]: If it already works, why change it?

Comment (JunTaoLuo) [inline samples/RoutingSample.Web/RoutingSample.Web.csproj]: We don't want samples bringing in everything in common.props such as the sdk, signing, ...

**Comment by dougbu** [comment]
@pranavkm 
> Revive the two API check files 

Why? The projects target only .NET Standard 2.0, never .NET Framework AnyWhere.AnyHow.

**Comment by pranavkm** [comment]
Yup, shouldn't need those.

File: samples/RoutingSample.Web/RoutingSample.Web.csproj (hunk 2 above)
Comment (ryanbrandenburg) [inline samples/RoutingSample.Web/RoutingSample.Web.csproj]: This sample is referenced by the FunctionalTests, which complain if it's not strongnamed when I replace common.props with dependencies.props. I vote that we leave it as is to get this PR through and unblock the PR's below me, and we can file a bug to not reference common.props

**Pull request merged**